src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from '@/components/leaderboard/TopThreeCard/style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  state: String;
+  rank: String;
+  name: String;
+  points: number;
+  image: String;
+}
+const positionNames = ['first', 'second', 'third'];
+
+const TopUserCard = ({ position, state, rank, name, points, image }: UserCardProps) => {
+  const trimName = name.length >= 25 ? `${name.substring(0, 24)}...` : name;
+  return (
+    <div className={`${styles.leaderboardCard} ${styles[positionNames[position - 1]]}`}>
+      <div className={styles.cardLeft}>{position}</div>
+      <div className={styles.cardRight}>
+        <img className={styles.profileImage} src={image} alt="profile-pic" />
```

Comment:
```suggestion
        <img className={styles.profileImage} src={image} alt="User Profile Image" />
```
One usage of alt tags is for screen readers and are meant to be human readable text